templates/navigation/fields/field--field-txex-ref-rev-para-01.html.twig

```diff
@@ -0,0 +1,7 @@
+<div class="main-menu__extra-content">
```

Comment:
Using this theme suggestion may be problematic because I fear that it is too general. Is there a more specific one (for only the menu_content_item entity) that we could use? Perhaps there is an issue i the module, or we can add a suggestion in preprocess.

Reply:
The only way we can get more specific (with the suggestions provided) is to add the view mode to the name. That doesn't really affect your concern though.

I figured this would be fine since there's no reason you'd add this field to anything else on the site.

Reply:
Sorted this in zoom